### PR TITLE
New pulsedot watchface

### PR DESCRIPTION
**Pull request by eLtMosen**
Featuring animations and double tap area to invert the black/white colour scheme.
Short demo video:
[https://mosushi.de/misc/asteroidos/pulsedot-new.mp4](https://mosushi.de/misc/asteroidos/pulsedot-new.mp4)

### CODE REVIEW

File: pulsedot/usr/share/asteroid-launcher/watchfaces/pulsedot.qml [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+                    animateColor7t.start()
+                    secondCanvas.colorctx = Qt.rgba(0, 0, 0, 0.6);
+                    secondCanvas.requestPaint();
+                    shrinkH.start()
+                    shrinkW.start()
+                    circleDoubleClickCover.toggle = 1
+                }
+            }
+        }
+        PropertyAnimation {id: growH; target: circleDoubleClickCover; properties: "height"; to: parent.width*0.9; duration: 0}
```
Comment (FlorentRevest) [inline pulsedot/usr/share/asteroid-launcher/watchfaces/pulsedot.qml:138]: Have you considered using a ParallelAnimation ? https://doc.qt.io/qt-5/qml-qtquick-parallelanimation.html basically you can "pack" them and synchronize their starts

**Review by FlorentRevest — COMMENTED** [review]
Could you also add it to the README ? Along with license of fonts ?

File: pulsedot/usr/share/asteroid-launcher/watchfaces/pulsedot.qml (hunk 1 above)
Comment (eLtMosen) [reply pulsedot/usr/share/asteroid-launcher/watchfaces/pulsedot.qml:138]: Not as of yet. 
But thanks for the hint. 
Indeed the animation department looks a bit cleaner too now.

**Comment by eLtMosen** [comment]
> Could you also add it to the README ? Along with license of fonts ?

ah, sad.
I have no means of doing a screenshot on a square watch since i gifted my dory to a friend.
Shall i just leave the space empty and ignore the inconsitancy?

**Comment by FlorentRevest** [comment]
Thanks! Great addition, it looks really good :D

**Pull request merged**